### PR TITLE
Add Portuguese translation for Tuples

### PR DESCRIPTION
**Pull request by LucasSantiag**
Tuples translation to portuguese

### CODE REVIEW

**Comment by eed3si9n** [comment]
@LucasSantiag Thanks for the contribution!

I can't review this myself, so could you find someone who can? /cc @marcospereira, @fforbeck

**Comment by dwijnand** [comment]
perhaps @renatocaval, too

File: _pt-br/tour/tuples.md [hunk 1]
```diff
@@ -1,15 +1,78 @@
...
 ---
 
-(this section of the tour has not been translated yet. pull request
-with translation welcome!)
\ No newline at end of file
+Em Scala, uma tupla é um valor que contém um número fixo de elementos, cada um com tipos distintos e as tuplas são imutaveis.
+
+Tuplas são sobretudo úteis para retornar múltiplos valores de um método.
+
+Uma Tupla com dois lementos pode ser criada dessa forma:
```
Comment (fforbeck) [inline _pt-br/tour/tuples.md]: lementos = elementos

File: _pt-br/tour/tuples.md [hunk 2]
```diff
@@ -1,15 +1,78 @@
...
+
+```tut
+val ingrediente = ("Açucar" , 25)
+```
+
+Isto cria uma tupla contendo dois elementos um `String` e o outro `Int`.
+
+O tipo inferido de `ingrediente` é `(String, Int)`, que é uma forma abreviada para `Tuple2[String, Int]` .
+
+Para representar tuplas, Scala usa uma serie de classes: `Tuple2`, `Tuple3`, etc., até `Tuple22` . Cada classe tem tantos parâmetros quanto elementos
```
Comment (fforbeck) [inline _pt-br/tour/tuples.md]: ...quanto elementos.

File: _pt-br/tour/tuples.md [hunk 3]
```diff
@@ -1,15 +1,78 @@
...
+num: 6
 next-page: mixin-class-composition
 previous-page: traits
 language: pt-br
 ---
 
-(this section of the tour has not been translated yet. pull request
-with translation welcome!)
\ No newline at end of file
+Em Scala, uma tupla é um valor que contém um número fixo de elementos, cada um com tipos distintos e as tuplas são imutaveis.
```
Comment (marcospereira) [inline _pt-br/tour/tuples.md]:
```suggestion
Em Scala, uma tupla é um valor que contém um número fixo de elementos, cada um com tipos distintos e as tuplas são imutáveis.
```

File: _pt-br/tour/tuples.md [hunk 4]
```diff
@@ -1,15 +1,78 @@
...
+
+```tut
+val ingrediente = ("Açucar" , 25)
+```
+
+Isto cria uma tupla contendo dois elementos um `String` e o outro `Int`.
+
+O tipo inferido de `ingrediente` é `(String, Int)`, que é uma forma abreviada para `Tuple2[String, Int]` .
+
+Para representar tuplas, Scala usa uma serie de classes: `Tuple2`, `Tuple3`, etc., até `Tuple22` . Cada classe tem tantos parâmetros quanto elementos.
```
Comment (marcospereira) [inline _pt-br/tour/tuples.md]:
```suggestion
Para representar tuplas, Scala usa uma serie de classes: `Tuple2`, `Tuple3`, etc., até `Tuple22` . Cada classe tem tantos parâmetros de tipo quanto elementos.
```

File: _pt-br/tour/tuples.md [hunk 5]
```diff
@@ -1,15 +1,78 @@
...
+
+## Acessando os Elementos
+
+Uma maneira de acessar os elementos da tupla é pela sua respectiva posição. Os elementos individuais são nomeados _1 , _2 , e assim por diante.
```
Comment (marcospereira) [inline _pt-br/tour/tuples.md]:
```suggestion
Uma maneira de acessar os elementos da tupla é pela sua respectiva posição. Os elementos individuais são nomeados `_1` , `_2` , e assim por diante.
```

File: _pt-br/tour/tuples.md [hunk 6]
```diff
@@ -1,15 +1,78 @@
...
+
+```tut
+println(ingrediente._1) // Açucar
+println(ingrediente._2) // 25
+```
+
+## Correspondência de padrões em tuplas
+
+Uma tupla pode tambem ser desmembrada usando correspondência de padrões:
```
Comment (marcospereira) [inline _pt-br/tour/tuples.md]:
```suggestion
Uma tupla pode também ser desmembrada usando correspondência de padrões:
```

File: _pt-br/tour/tuples.md [hunk 7]
```diff
@@ -1,15 +1,78 @@
...
+
+```tut
+val (nome, quantidade) = ingrediente
+println(nome) // Açucar
+println(quantidade) // 25
+```
+
+Aqui `nome` é um tipo inferido `String` e `quantidade` é um tipo inferido `Int` .
```
Comment (marcospereira) [inline _pt-br/tour/tuples.md]:
```suggestion
Aqui o tipo inferido para `nome` é `String` e para `quantidade` o tipo inferido é `Int`.
```

File: _pt-br/tour/tuples.md [hunk 8]
```diff
@@ -1,15 +1,78 @@
...
+  List(("Mercurio", 57.9), ("Vénus", 108.2), ("Terra", 149.6),
+       ("Marte", 227.9), ("Júpiter", 778.3))
+planetas.foreach{
+  case ("Terra", distancia) =>
+    println(s"Nosso planeta está a $distancia milhões de kilometros do sol")
+  case _ =>
+}
+```
+
+ou, um exemplo com `for` :
```
Comment (marcospereira) [inline _pt-br/tour/tuples.md]:
```suggestion
Ou, um exemplo com `for` :
```

File: _pt-br/tour/tuples.md [hunk 9]
```diff
@@ -1,15 +1,78 @@
...
+
+```tut
+val numPars = List((2, 5), (3, -7), (20, 56))
+for ((a, b) <- numPars) {
+  println(a * b)
+}
+```
+
+## Tuplas e classes case
```
Comment (marcospereira) [inline _pt-br/tour/tuples.md]:
```suggestion
## Tuplas e case classes
```

Comment (marcospereira) [reply _pt-br/tour/tuples.md]: I think it is better to not _translate_ `case classes` here.

File: _pt-br/tour/tuples.md [hunk 10]
```diff
@@ -1,15 +1,78 @@
...
+
+Desenvolvedores às vezes acham dificil escolher entre tuplas e classes case. Classes case tem elementos nomeados, e esses podem melhorar a leitura de alguns tipos de códigos. No planeta exemplo abaixo, nós poderiamos definir uma `case class Planeta(nome: String, distancia: Double)` ao invés de usar tuplas.
```
Comment (marcospereira) [inline _pt-br/tour/tuples.md]:
```suggestion
Desenvolvedores às vezes acham dificil escolher entre tuplas e case classes. Case classes têm elementos nomeados, e esses podem melhorar a leitura de alguns tipos de códigos. No exemplo dos planetas acima, nós poderiamos definir uma `case class Planeta(nome: String, distancia: Double)` ao invés de usar tuplas.
```

File: _pt-br/tour/tuples.md [hunk 11]
```diff
@@ -1,15 +1,78 @@
...
+
+Outro exemplo de correspondência de padrões em uma tupla:
+
+```tut
+val planetas =
+  List(("Mercurio", 57.9), ("Vénus", 108.2), ("Terra", 149.6),
+       ("Marte", 227.9), ("Júpiter", 778.3))
+planetas.foreach{
+  case ("Terra", distancia) =>
+    println(s"Nosso planeta está a $distancia milhões de kilometros do sol")
```
Comment (renatocaval) [inline _pt-br/tour/tuples.md]:
```suggestion
 println(s"Nosso planeta está a $distancia milhões de quilômetros do sol")
```

**Comment by LucasSantiag** [comment]
Thanks for all the suggestions, next time I will pay attention to these little details.

**Comment by eed3si9n** [comment]
Thanks @LucasSantiag and the reviewers!

**Pull request merged**